IoTEnterpriseDefault/CS/SmartDisplay/Utils/UI/ViewLifetimeControl.cs

```diff
@@ -0,0 +1,260 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+
+// The objects defined here demonstrate how to make sure each of the views created remains alive as long as 
+// the app needs them, but only when they're being used by the app or the user. 
+//
+// The Consolidated event is fired when a view stops being visible separately from other views. Common cases where this will occur
+// is when the view falls out of the list of recently used apps, or when the user performs the close gesture on the view.
+// This is a good time to close the view, provided the app isn't trying to show the view at the same time. This event
+// is fired on the thread of the view that becomes consolidated.
+//
+// Each view lives on its own thread, so concurrency control is necessary. Certain objects may be bound to UI on given threads. 
+// Properties of those objects should only be updated on that UI thread.
+
+using System;
+using System.ComponentModel;
+using Windows.UI.Core;
+using Windows.UI.ViewManagement;
+
+/// <summary>
+/// Based on the Multi-View Universal Windows Platform (UWP) sample
+/// </summary>
+namespace SmartDisplay.Utils.UI
+{
+    /// <summary>
+    /// A custom event that fires whenever the secondary view is ready to be closed. You should
+    /// clean up any state (including deregistering for events) then close the window in this handler
+    /// </summary>
+    /// <param name="sender"></param>
+    /// <param name="e"></param>
+    public delegate void ViewReleasedHandler(object sender, EventArgs e);
+
+    /// <summary>
+    /// A ViewLifetimeControl is instantiated for every secondary view. ViewLifetimeControl's reference count
+    /// keeps track of when the secondary view thinks it's in usem and when the main view is interacting with the secondary view (about to show
+    /// it to the user, etc.) When the reference count drops to zero, the secondary view is closed.
+    /// </summary>
+    public sealed class ViewLifetimeControl : INotifyPropertyChanged
+    {
+        // Dispatcher for this view. Kept here for sending messages between this view and the main view.
+        public CoreDispatcher Dispatcher { get; private set; }
+
+        // Each view has a unique Id, found using the ApplicationView.Id property or
+        // ApplicationView.GetApplicationViewIdForEnterpriseWindow method. This id is used in all of the ApplicationViewSwitcher
```

Comment:
Change GetApplicationViewIdForEnterpriseWindow back to GetApplicationViewIdForCoreWindow.  This is an actual API method name not a reference to IoT Core.

Reply:
Made the changes